### DISCUSSION TITLE
Error in updating an article with tag input

### DISCUSSION THREAD

**Discussion by Amazighettal** [area17/twill#2022] [Q&A]
Hello, i have this error win updating an article with tag input 
preg_match() expects parameter 2 to be string, array given in parent::aftersave()
Thanks

**Comment by haringsrob** [comment]
hey @Amazighettal are you on the latest version?

**Reply by Amazighettal** [reply]
hello @haringsrob I'm using area17/twill : "^2.8" 

file: "/var/www/html/project/vendor/area17/twill/src/Repositories/Behaviors/HandleTags.php"
function: "preg_match" line: 14

**Reply by haringsrob** [reply]
Hey @Amazighettal this should be resolved by now, you can update to the latest Twill version: https://github.com/area17/twill/pull/1993

**Reply by Amazighettal** [reply]
thank you

**Comment by Amazighettal** [comment]
Hi, I have an error cors CORS policy in image cropping 
<img width="1437" alt="Capture d’écran 2023-01-16 à 15 48 07" src="https://user-images.githubusercontent.com/118449881/212706201-5a1a28da-7349-4f22-9eea-8e92a1598aaf.png">
